sunpy/map/mapbase.py

```diff
@@ -133,6 +136,33 @@ class GenericMap(NDData):
         Additional keyword arguments are passed to `~astropy.nddata.NDData`
         init.
 
+
+    Table of Methods and Their Behavior with Dask Arrays
+    -----------------------------------------------------
+
+    +-------------------+--------------------------------+
+    | Method            | Compatibility with Dask Arrays |
```

Comment:
```suggestion
    | Method            | Preserve laziness with Dask Arrays |
```
The table approach does make changing the titles and the width of the columns a pain.